utils/fo-installdeps

```diff
@@ -110,13 +110,13 @@ if [ $BUILDTIME ]; then
               lenny|hardy) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.3;;
               squeeze) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               wheezy) apt-get $YesOpt install $YesOpt postgresql-server-dev-9.1;;
-              sid) apt-get  $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               lucid|natty) apt-get $YesOpt install postgresql-server-dev-8.4;;
               isadora) apt-get $YesOpt install postgresql-server-dev-8.4;;
               precise|oneiric|quantal|raring|saucy) apt-get $YesOpt install postgresql-server-dev-9.1;;
               trusty) apt-get $YesOpt install postgresql-server-dev-9.3;;
               jessie) apt-get $YesOpt install postgresql-server-dev-9.4;;
-              xenial) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              xenial|yakkety) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              stretch|buster|sid|zesty|artful) apt-get $YesOpt install postgresql-server-dev-9.6;;
```

Comment:
wel as i said it will not "break" - it will just require a new version of the PG libs - and that WILL happen - and it will happen in buster and ubuntu at the same time (within a few months)

you could use debootstrap (or several other tools) for that - but yea it's probably more trouble than it's worth - to be consistent then i would suggest to drop support for the non-LTS versions of ubuntu because it is no easier to support those on travis right?

also as i mention on my original PR - the entire hassle of version numbers could be avoided by requiring the metapackages like:
```
postgresql-server-dev-all
instead of
postgresql-server-dev-9.5
or
postgresql-server-dev-9.6

postgresql php php-pgsql php-cli php-curl php-xml
instead of
postgresql-9.5 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl
or
postgresql-9.6 libapache2-mod-php7.0 php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-xml;;
```

just some ideas to lessen the maintenance load - fwiw - thi sis a nice feature but very few upstreams manage a script like this - they usually just post the dependencies in the README and let it to the user and a ./configure script to prepare the environment